drivers/storage/portworx/preflight.go

```diff
@@ -358,9 +360,18 @@ func (u *preFlightPortworx) processNodesChecks(recorder record.EventRecorder, st
 				k8sutil.InfoEvent(recorder, u.cluster, util.PassPreFlight, msg)
 				continue
 			}
+
+			passed = false // pre-flight status check failed, keep going for logging
+
+			if check.Result == "fatal" {
```

Comment:
nit: Let's use a capitalized constant. Also, can we declare this constant above?